odoo/addons/base/tests/test_qweb.py

```diff
@@ -482,8 +481,10 @@ def test_render_dynamic_xml_with_code_error(self):
             """
         })
         error_msg = "Can't convert 'int' object to str implicitly"
```

Comment:
this line is not needed anymore, right?

Reply:
@mart-e  
Yes, you're right. In fact, I noticed this PR is not required anymore, because the same was don on https://github.com/odoo/odoo/commit/cc0baa086afc35cc2ef9eca72c3d7abba43550da, so I'll close it.